.gitlab/e2e.yml

```diff
@@ -194,6 +194,21 @@ new-e2e-agent-subcommands-main:
   # Temporary, until we manage to stabilize those tests.
   allow_failure: true
 
+new-e2e-language-detection-dev:
+  extends: .new_e2e_template
+  rules: !reference [ .on_dev_branch_manual ]
+  needs: [ ]
+  variables:
+    TARGETS: ./language-detection
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
We'll soon enforce adding a `TEAM` variable to jobs, used to tag aws resources. This will help with cost monitoring
```suggestion
    TARGETS: ./language-detection
    TEAM: process
```

Reply:
Done!